### PR TITLE
chore: ⬆️ add logging about filtering to DAS handler [sc-95142]

### PR DESCRIPTION
**Pull request by ad-m-ss**
## Description

Currently SQL Server is dropping data and not even writing it to S3. I assume the data is filtered out. Currently there is no information about this in CW, so let's add logs.

## Type of change

-   [ ] Refactor (non-breaking change that improves codebase).
-   [ ] Bug fix (non-breaking change that fixes an issue).
-   [ ] New feature (non-breaking change that adds functionality).
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
-   [ ] Chore (none of the above, but still important)

## Related tickets

-   Story sc-XXXXX

## Checklists

-   [ ] No backend changes needed.
-   [ ] All related backend changes are ready.
    -   [links to related PRs]

### CODE REVIEW

File: das/deploy.sh [hunk 1]
```diff
@@ -19,3 +19,5 @@ python update.py "$BUCKET" "$PREFIX/deployment-package.zip"
 # Upload files
 aws s3 cp ./SelectStarDAS.json "s3://$BUCKET/$PREFIX/" --acl public-read
 aws s3 cp ./das-firehose-log-process/handler.zip "s3://$BUCKET/$PREFIX/deployment-package.zip" --acl public-read
+
+# aws lambda update-function-code --function-name SelectStarDASIntegrationS-ProcessKinesisDataLambda-... --zip-file fileb://das-firehose-log-process/handler.zip --publish
```
Comment (verdan) [inline das/deploy.sh:23]: @ad-m-ss is this intentional?

Comment (ad-m-ss) [reply das/deploy.sh:23]: Yes, I found it handy to make a hacky partial update, but it's error prone, so I just left it here undocumented for ppl who understand it anyway.

Comment (verdan) [reply das/deploy.sh:23]: can we at least add a comment on why this is here?

Comment (ad-m-ss) [reply das/deploy.sh:24]: Added

**Review by verdan — APPROVED** [review]

**Pull request merged**